server/services/cms.js

```diff
@@ -32,14 +38,25 @@ class CmsService {
     return tagResult;
   }
 
-  async getTag(establishmentName, tagId) {
-    const lookupData = await this.#cmsApi.lookupTag(establishmentName, tagId);
+  async getSeries(establishmentName, id, location) {
+    const result = await this.#cmsApi.get(
+      new SeriesPageQuery(establishmentName, id),
+    );
+    if (result?.name) return result;
+    const tagResult = await this.#cmsApi.get(
+      new SeriesHeaderPageQuery(location),
+    );
+    return tagResult;
+  }
+
+  async getTag(establishmentName, id) {
+    const lookupData = await this.#cmsApi.lookupTag(establishmentName, id);
     const { type, uuid, location } = lookupData;
     switch (type) {
       case 'taxonomy_term--tags':
         return this.getSecondaryTag(establishmentName, uuid, location);
       case 'taxonomy_term--series':
-        return null; // TODO
+        return this.getSeries(establishmentName, id, location);
```

Comment:
we could possibly throw from default now if we come across some type of content we wouldn't expect to serve?

Reply:
Agreed, but we still have to cater for categories atm.